userspace/libscap/scap_procs.c

```diff
@@ -960,42 +977,25 @@ int32_t scap_proc_scan_proc_dir(scap_t* handle, char* procdirname, int parenttid
 		// if tid_to_scan is set we assume is a runtime lookup so no
```

Comment:
This comment references a parameter that you removed...looks like the logic to determine if it's a runtime lookup is just no longer present. Might want to just remove the comment or (ideally) add a more helpful comment explaining the existing logic.